src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -488,14 +506,20 @@ private void getNextBatch(String requestId, Request request, boolean firstBatch,
       try {
         for (JsonEvent jsonEvent : batch) {
           JsonObject value = jsonEvent.objectValue();
-          String instanceId = value.getString(ENRICHED_INSTANCE_ID);
-          Object itemsandholdingsfields = value.getValue(RecordMetadataManager.ITEMS_AND_HOLDINGS_FIELDS);
+          String instanceId = value.getString(INSTANCE_ID_FIELD_NAME);
+          Object itemsandholdingsfields = value;
```

Comment:
Why this variable is Object type if value-variable is JsonObject type?

Reply:
Yes, agree. Moreover, this variable is redundant.